weedcoco/index/thumbnailing.py

```diff
@@ -0,0 +1,20 @@
+import argparse
+import os
+import pathlib
+from PIL import Image
+
+ap = argparse.ArgumentParser(description=__doc__)
+ap.add_argument("--thumbnails-dir", type=pathlib.Path, required=True)
+ap.add_argument("--repository-dir", type=pathlib.Path, required=True)
+args = ap.parse_args()
+
+THUMBNAIL_SIZE = (300, 300)
```

Comment:
This looks good to me; it does the job 👍 

One question I wonder about here is the behaviour of image.thumbnail: I know in our previous thumbnail implementation we ended up with thumbnails in different aspect ratios. from a front end visual perspective this was kind of weird looking. Something to look out for when testing this :)